src/api/bible.ts

```diff
@@ -1,9 +1,9 @@
 import { Module } from "../native";
-import { BibleReference, BibleVersion } from "../types";
+import { BibleReferenceBase, BibleVersion } from "../types";
 
 export const BibleAPI = {
   /**
-   * Retrieves a list of Bible versions available for a specified language code (like "en").
+   * Retrieves a list of Bible versions available for a specified language code (like "eng").
```

Comment:
Is `eng` a language code or a language tag?

```suggestion
   * Retrieves a list of Bible versions available for a specified language tag (like "eng").
```